ftva_lab_data/views.py

```diff
@@ -157,6 +157,12 @@ def render_search_results_table(request: HttpRequest) -> HttpResponse:
 
     paginator = Paginator(items, 10)
     page_obj = paginator.get_page(page)
+    # Convert elided page range to list to allow multiple iterations in template
+    elided_page_range = list(
+        paginator.get_elided_page_range(
+            number=page_obj.number, on_each_side=2, on_ends=1
```

Comment:
I think we should offer more pages `on_each_side`, maybe 4 or even 5, to allow for quicker navigation through large result sets.  I agree with `on_ends=1`, I don't see value to expanding that.